### ISSUE TITLE
unable to increase android target version in 1.9.5-beta1

### ISSUE THREAD

**Issue by WimpyMistake** [czyzby/gdx-setup#43]
As per title, im using gdx-setup 1.9.5-beta1.
In Advanced tab:
- unable to increase Target Android API to more than 23 (current is 25)
- and also this values didn't get remembered (i set values first then exit the app, then reenter the app, the values return back to original values) not sure if its intended though.
  - Java Version - changed to 1.7 then return to 1.6
  - Android plugin - changed to 2.2.3 then return to 1.5.0
  - roboVM version - changed to 2.3.0 then return to 2.1.0

changing build tools version and gwt version gets remembered anyway

**Comment by czyzby** [comment]
Yeah, not every single setting is saved in user preferences - we designed it this way, since some preferences might vary from project to project and since most are not limited by a list of valid values (like GWT) or numeric ranges (like Android SDK). Some users might save invalid values in their preferences and use them for all projects by mistake.

Anyway, I updated the accepted Android SDK versions range.
